### DISCUSSION TITLE
How to render livewire component inside modal?

### DISCUSSION THREAD

**Discussion by Sitramas** [filamentphp/filament#13704] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

How to render full size livewire component inside modal? Example bellow not working, because modalContent argument should be Htmlable...
```
TextInput::make('input')
   ->suffixAction(
       Action::make('action')
          ->icon('heroicon')
          ->modalContent(function () {
                    return Livewire::make('myComponent');
           })
   )
```
Thanks for help.

**Comment by awcodes** [comment, marked as answer]
ModalContent should return a view. Then in your view you can render your livewire component.
